### DISCUSSION TITLE
LDAP as a backend for users

### DISCUSSION THREAD

**Discussion by zwarag** [pocketbase/pocketbase#4132] [Q&A]
I'm part of a hackerspace and we are currently looking into solutions for backends. I stumbled onto pocketbase and it looks great.
I was wondering if it is possible to use LDAP as a Backend for the Users. We use OpenLDAP to manage our users and would like to keep the management of the users there as there are other services which are build onto that.
However, it looks like only Oauth is supported right?

![image](https://github.com/pocketbase/pocketbase/assets/2607973/bf976c58-0346-490c-8a40-ac3f68cac98d)

**Comment by MrGoodbits** [comment]
I am also interested in LDAP.  It is more capable for our internal needs.
